### PR TITLE
Fix existing command to be not alias-able

### LINKED ISSUE #191: [PE-D] Alias commands allow you to change re-existing commands
I changed help to list, so now I cannot even use the help command at all?

Similarly (without the previous command), I change alias-add to list, so now i can't use list command.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: kieron560/ped#5

### PR DESCRIPTION
**Pull request by tanboonji**
Closes #191

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -154,4 +154,40 @@ public boolean isValidCommandToAlias(String userInput) {
...
+
+        case TagCommand.COMMAND_WORD:
+
+        case SelectCommand.COMMAND_WORD:
+            return false;
+
+        default:
+            return true;
+        }
+    }
```
Comment (yaowei-soc) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:191]: Should this utility function be in `ParserUtil.java` instead?

**Review by yaowei-soc — CHANGES_REQUESTED** [review]
Changes LGTM but with possible code quality issue

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 1 above)
Comment (tanboonji) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java:191]: I have thought of putting it inside `ParserUtil.java`, but the reason I put in `AddressBookParser.java` is that the logic to check if a user input of the alias and command is valid to be aliased requires the dependency of all other command classes, and the most suitable class to put this logic in where the class has dependency of all other command classes would be the `AddressBookParser.java`.

**Review by oeiyiping — APPROVED** [review]
Aside from the matter still open for discussion, LGTM.

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 1 above)
Comment (yaowei-soc) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java:191]: Agree that this adding to `ParserUtil.java` would create a cycling dependency.

**Review by yaowei-soc — APPROVED** [review]

**Pull request merged**